tests/Presenters/RuntimeEnvironmentsPresenter.phpt

```diff
@@ -26,7 +26,7 @@ class TestRuntimeEnvironmentsPresenter extends Tester\TestCase
   private $presenterPath = "V1:RuntimeEnvironments";
 
   /** @var RuntimeEnvironments */
-  protected $hardwareGroups;
+  protected $runtimeEnvironments;
```

Comment:
Context? Is that just a fix of bad naming?

Reply:
Just a fix which I ran into, while checking this presenter, somehow it worked without this variable.